pages/linux/ss.md

```diff
@@ -2,26 +2,26 @@
 
 > Utility to investigate sockets.
 
-- Dump all TCP/UDP/RAW/UNIX sockets:
+- Show all TCP/UDP/RAW/UNIX sockets:
 
 `ss -a {{-t/-u/-w/-x}}`
```

Comment:
This whole example bothers me tbh. I'd have preferred to describe 3 or so of these in other examples that also introduce filters, but I can't come up with a good Unix example so all I can manage is tcp and udp